packages/neuron-wallet/tests/controllers/offline-sign.test.ts

```diff
@@ -224,15 +225,21 @@ describe('OfflineSignController', () => {
           canceled: true
         })
 
-        expect(offlineSignController.exportTransactionAsJSON({} as any)).rejects.toThrow()
+        const res = await offlineSignController.exportTransactionAsJSON({} as any)
+        expect(res).toEqual({
+          status: ResponseCode.Success
+        })
       })
 
       it('throws if file path is not provider', async () => {
         stubbedElectronShowSaveDialog.mockReturnValue({
           canceled: false
         })
 
-        expect(offlineSignController.exportTransactionAsJSON({} as any)).rejects.toThrow()
+        const res = await offlineSignController.exportTransactionAsJSON({} as any)
```

Comment:
Test case should be renamed(https://github.com/nervosnetwork/neuron/pull/2406/files#diff-fc0481ee942e04139944b19f957b2ad9b1bd24867540ca31f29b12add07c1b20R234).